js/common/lib/env.ts

```diff
@@ -143,9 +143,44 @@ export declare namespace Env {
        */
       ondata?: (data: WebGpuProfilingData) => void;
     };
+    /**
+     * Set or get the power preference.
+     *
+     * Setting this property only has effect before the first WebGPU inference session is created. The value will be
```

Comment:
Can we use WebGpuExecutionProviderOption to host both powerPreference and forceFallbackAdapter, instead of env variables?

Reply:
WebGpuExecutionProviderOption is designed to put per-session options and ort.env is used for global properties.